pkg/volume/azure_dd/azure_common.go

```diff
@@ -64,7 +64,8 @@ var (
 		string(api.AzureManagedDisk))
 
 	// only for Windows node
```

Comment:
Wouldn't it make sense to move the Windows-specific code to the Windows-specific file, azure_common_windows.go?

Reply:
Yes, done. Thanks again!